### PR TITLE
Update global search for experiment level

### PR DESCRIPTION
**Pull request by jbargu**
https://biosistemika.atlassian.net/browse/SCI-33

### CODE REVIEW

File: app/models/experiment.rb [hunk 1]
```diff
@@ -23,6 +23,47 @@ class Experiment < ActiveRecord::Base
...
+    if include_archived
+      new_query =
+        Experiment
+          .where(project: project_ids)
+          .where_attributes_like([:name, :description], a_query)
+    else
+      new_query =
+        Experiment
+          .is_archived(false)
+          .where(project: project_ids)
```
Comment (Ducz0r) [inline app/models/experiment.rb]: Nice, I didn't know you can write the `.id IN (?)` condition this elegantly!

File: app/models/my_module_group.rb [hunk 2]
```diff
@@ -30,8 +31,8 @@ def self.search(user, include_archived, query = nil, page = 1)
     new_query = MyModuleGroup
       .distinct
       .joins(:experiment)
```
Comment (Ducz0r) [inline app/models/my_module_group.rb]: I don't think this join is needed here any more?

**Comment by Ducz0r** [comment]
I've reviewed the MR, and it looks good (with the exception of 1 remark I made in the comments). Jolly good work! :+1: 
I sense somebody's been busy using Rubocop lately.

File: app/models/my_module_group.rb (hunk 2 above)
Comment (jbargu) [inline app/models/my_module_group.rb]: Good catch!

**Comment by jbargu** [comment]
Is it that obvious? :grin: 
Nah, I actually haven't set it up yet. The above lines just looked hideous in terminal, so I decided to fix  them along. You could say the above lines got TERMINAL treatment. :sunglasses:

**Pull request merged**